backends/tfhe-cuda-backend/cuda/include/integer/integer_utilities.h

```diff
@@ -5578,6 +5578,91 @@ template <typename Torus> struct int_signed_scalar_div_mem {
   }
 };
 
+template <typename Torus> struct unsigned_scalar_div_rem_buffer {
```

Comment:
We also have to be consistent with the struct naming. Since this is a struct that relates to the integer scalar division it should start with `int_`. For instance, the signed version name is `int_signed_scalar_div_mem`.